### DISCUSSION TITLE
enable old controlnet extension

### DISCUSSION THREAD

**Discussion by JGrant15** [vladmandic/sdnext#3076] [Q&A] [answered]
Hi, I was using an old version of SD Next ~ late Oct 2023, today i updated the repo only to realise that Control is now a thing and the old controlnet-webui extension is disabled, my problem is I built an app based around making api calls to SD Next which also uses control net, these api calls no longer work. I have tried rolling back both sd-next and the extension but I'm getting a bunch of errors, is there a way to renable the extension or is control the only option now? My app is been used at an event on Wednesday and I don't have time to rewrite everything, any help would be greatly appreciated

**Comment by vladmandic** [comment, marked as answer]
switch in settings to backend=original and restart the server.

**Reply by JGrant15** [reply]
Yes that done the trick thank you for the quick response

**Comment by brianjz** [comment]
Jumping on there with a related question, is there a way to use the new Control capabilities via the API?
